src/Corporation/Actions.ts

```diff
@@ -207,8 +207,8 @@ export function sellMaterial(material: Material, amount: string, price: string):
   try {
     if (temp.includes("MP")) throw "Only one reference to MP is allowed in sell price.";
     temp = eval?.(temp);
-  } catch (e) {
-    throw new Error("Invalid value or expression for sell price field: " + e);
+  } catch (error) {
+    throw new Error(`Invalid value or expression for sell price field: ${String(error)}`);
```

Comment:
Here and everywhere you are making this type of change:
I think the better change would be
```suggestion
    throw new Error("Invalid value or expression for sell price field", {cause: error});
```
Here is what the difference looks like to me:
![image](https://github.com/user-attachments/assets/d0d633bd-11c0-4da7-b761-eefbda336fe8)

This will be a lot of changes (although mechanical), so if you don't think it's right or you don't want to do it, that's fine with me.

Reply:
Isn't this silly? We're throwing a string, not an error above. Following the TSC4.4 unknown in catch clause, this particular scenario should be type-checked.

If it's an error, print error.message, if it's a string, concatenate it to the Message in the catch block.